compiler-core/src/type_/tests/warnings.rs

```diff
@@ -1140,16 +1140,12 @@ fn unused_alias_warning_test() {
 
 #[test]
 fn used_type_with_import_alias_no_warning_test() {
-    assert_warnings_with_imports!(
-        ("gleam/foo", "pub type A");
+    assert_no_warnings!(
+        ("gleam", "foo", "pub type A"),
```

Comment:
Here the module is called `foo` but the test is importing `gleam/foo`, so I think the is a compile error rather than a test that replicates the problem. If the fix is commented out does this test fail?

Reply:
My bad, I thought the "gleam" in `("gleam", "foo", "pub type A")` is part of the import path.
I fixed the this test and one other place where I made the same mistake. Now the test fails without the fix.